chia/wallet/wallet_state_manager.py

```diff
@@ -1472,26 +1472,23 @@ async def coin_added(
             wallet_id,
             wallet_type,
         )
-        farmer_reward = False
-        pool_reward = False
-        if self.is_farmer_reward(height, coin):
-            farmer_reward = True
-        elif self.is_pool_reward(height, coin):
-            pool_reward = True
-
-        farm_reward = False
+
+        if self.is_pool_reward(height, coin):
+            tx_type = TransactionType.COINBASE_REWARD
+        elif self.is_farmer_reward(height, coin):
+            tx_type = TransactionType.FEE_REWARD
+        else:
+            tx_type = TransactionType.INCOMING_TX
+
+        coinbase = tx_type != TransactionType.INCOMING_TX
```

Comment:
it seems a bit risky to rely on no other enum values being added to this enum (and becoming possibilities here).
It's not a big deal, but I think it would be slightly easier to read and slightly more robust to say:
```
coinbase = tx_type in [TransactionType.FEE_REWARD, TransactionType.COINBASE_REWARD]
```

Reply:
Good point, added 3822fc51b97594afc4dc7dfd1906761b6a5cced4.